service/discovery/azure/compute.go

```diff
@@ -99,12 +129,20 @@ func (d *azureComputeDiscovery) List() (list []voc.IsResource, err error) {
 		list = append(list, s)
 	}
 
-	// Discover Load Balancer
-	// TODO Client to get load balancer
+	return list, err
+}
+
+// Discover Load Balancer
+func (d *azureComputeDiscovery) discoverLoadBalancer() ([]voc.IsResource, error) {
```

Comment:
I have moved load balancers out of the `Compute` resource, so a separate discoverer would be more appropriate.